sdk/spring/azure-identity-spring/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
-## 1.11.0-beta.1 (Unreleased)
+## 1.11.0 (2021-11-24)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Regular updates for Azure SDK dependency versions. This release is compatible with Spring Boot 2.5.0 - 2.5.4.
```

Comment:
To keep the same style of all changelog entries, I prefer write like this:

```
## 1.11.0 (2021-11-24)
This release is compatible with Spring Boot 2.5.0 - 2.5.4.

### Dependency Upgrades
- Regular updates for Azure SDK dependency versions.

```

Reply:
fair enough